panther_docking/README.md

```diff
@@ -13,37 +13,35 @@ The package contains a `PantherChargingDock` plugin for the [opennav_docking](ht
 
 ## ROS Nodes
 
+- `DockPosePublisherNode`: A node listens to `tf` and republishes position of `dock_pose` in the fixed frame.
 - `PantherChargingDock`:  A plugin for a Panther robot what is responsible for a charger service.
 
-### PantherChargingDock
+### DockPosePublisherNode
 
 #### Publishes
 
 - `docking/dock_pose` [*geometry_msgs/PoseStamped*]: An offset dock pose.
-- `docking/staging_pose` [*geometry_msgs/PoseStamped*]: An offset staging pose next to a charging station.
 
 #### Subscribers
 
-- `battery/charging_status` [*panther_msgs/ChargingStatus*]: A charging status of Panther robot.
-- `hardware/io_state` [*panther_msgs/IOState*]: States of GPIOs of Panther robot. The `PantherChargingDock` subscribes this topic to check if a robot charges.
+- `tf` [*tf2_msgs/TFMessage*]: Tf tree with a detected dock transform.
+
+### PantherChargingDock
+
+#### Publishes
 
-#### Service Clients
+- `docking/staging_pose` [*geometry_msgs/PoseStamped*]: An offset staging pose next to a charging station.
 
-- `hardware/charger_enable` [*std_srvs/SetBoot*]: This service client enables charging in a robot.
+#### Subscribers
+
+- `docking/dock_pose` [*geometry_msgs/PoseStamped*]: An offset dock pose.
 
 #### Parameters
 
-- `~<dock_name>.base_frame` [*string*, default: **base_link**]: A base frame id of a robot.
-- `~<dock_name>.external_detection_timeout` [*double*, default: **0.2**]: A timeout in seconds for looking up a transformation from an april tag of a dock to a base frame id.
-- `~<dock_name>.external_detection_translation_x` [*double*, default: **0.0**]: A translation over an X axis between a detected frame and a dock pose.
-- `~<dock_name>.external_detection_translation_y` [*double*, default: **0.0**]: A translation over an Y axis between a detected frame and a dock pose.
-- `~<dock_name>.external_detection_translation_x` [*double*, default: **0.0**]: A translation over a Z axis between a detected frame and a dock pose.
-- `~<dock_name>.external_detection_rotation_roll` [*double*, default: **0.0**]: A rotation over an X axis between a detected frame and a dock pose.
-- `~<dock_name>.external_detection_rotation_pitch` [*double*, default: **0.0**]: A rotation over an Y axis between a detected frame and a dock pose.
-- `~<dock_name>.external_detection_rotation_yaw` [*double*, default: **0.0**]: A rotation over a Z axis between a detected frame and a dock pose.
+- `~panther_version` [*double*, default: **1.21**]: A version of Panther robot.
+- `~base_frame` [*string*, default: **base_link**]: A base frame id of a robot.
+- `~external_detection_timeout` [*double*, default: **0.2**]: A timeout in seconds for looking up a transformation from an april tag of a dock to a base frame id.
```

Comment:
It was a bit messy... Sorry for that.

Actually there is no private parameters :D
The `<dock_name>` is added because the configuration of `docking_server` looks like this:
https://github.com/husarion/panther_ros/blob/ros2-docking-charging-dock/panther_docking/config/panther_docking_server.yaml#L17